CCP.Localization/persona.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Persona",
+  "type": "object",
+  "properties": {
+    "model": {
+      "type": "string",
+      "enum": [ "GPT-4o-mini" ]
+    },
+    "prompt": {
+      "type": "string"
+    }
+  },
+  "required": [ "model" ],
```

Comment:
I'm wondering if the prompt should be required as well? I think we will always need some kind of prompt, even if it is very simple.